tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -44,6 +44,9 @@ PointerVariableConstraint::PointerVariableConstraint(const Type *_Ty,
     vars.insert(K);
     CS.getOrCreateVar(K);
     K++;
+
+    if (tyToStr(Ty) == "struct __va_list_tag *")
```

Comment:
There should be some way in clang to test this without having to serialize it to string.